sdk/communication/azure-communication-networktraversal/src/samples/java/com/azure/communication/networktraversal/CreateAndIssueRelayCredentialsExample.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.networktraversal;
+
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.communication.identity.CommunicationIdentityClient;
+import com.azure.communication.identity.CommunicationIdentityClientBuilder;
+import com.azure.communication.networktraversal.models.CommunicationRelayConfiguration;
+import com.azure.communication.networktraversal.models.CommunicationIceServer;
+import java.util.List;
+
+public class CreateAndIssueRelayCredentialsExample {
```

Comment:
Add javadoc.

Reply:
This a sample class, I don't see usage of javadocs in samples for identity or chat. I can add it though.

Reply:
We should add javadoc for all samples as it would help the user understand what scenaio this sample is demonstrating.